src/legacy/store/GraphQLFragmentPointer.js

```diff
@@ -67,11 +67,11 @@ class GraphQLFragmentPointer {
     }
     invariant(
       typeof identifyingArgValue === 'string' || identifyingArgValue == null,
-      'GraphQLFragmentPointer: Value for the argument to `%s` on query `%s` ' +
-      'should be a string, but it was set to `%s`. Check that the value is a ' +
-      'string.',
-      query.getFieldName(),
+      'GraphQLFragmentPointer: On the root query `%s`, Value for the ' +
```

Comment:
nit: grammar/capitalization. Maybe 

    'GraphQLFragmentPointer: In query `%s` the value for argument `%s` must be a string, but it was set to `%s`. Check that the value is a string (note that this restriction applies to `query` arguments only).'